api/tests/unit/migrations/extract-person-contacts.js

```diff
@@ -37,246 +37,29 @@ exports['run does nothing if no facilities'] = function(test) {
-    test.equals(updatePlace.firstCall.args[0], 'a');
-    test.deepEqual(updatePlace.firstCall.args[1], { parent: 'b'});
-
-    // No contact created.
-    test.equals(createPerson.callCount, 0);
-
-    test.done();
-  });
-};
-
-exports['run still updates parent if facility have no contact'] = function(test) {
-  testParentUpdated(
-    test,
-    {
-      _id: 'a',
-      parent: { _id: 'b', contact: {}}
-    },
-    {
-      _id: 'a',
-    });
-};
-
-exports['run still updates parent if facility has migrated contact'] = function(test) {
-  testParentUpdated(test,
-    {
-      _id: 'a',
-      parent: { _id: 'b', contact: {}},
-      contact: { _id: 'd' }
-    },
-    {
-      _id: 'a',
-      contact: { _id: 'd' }
-    });
-};
-
-var testContactUpdated = function(test, doc, docWithoutContact) {
-  test.expect(12);
-
-  // Get the 3 levels of hierarchy
-  getView.onCall(0).callsArgWith(3, null, { rows: [ { id: 'a' }] });
-  getView.onCall(1).callsArgWith(3, null, { rows: [ ] });
-  getView.onCall(2).callsArgWith(3, null, { rows: [ ] });
-
-  // Get doc for parent update : no parent, so parent update is skipped.
-  getDoc.onCall(0).callsArgWith(1, null, doc);
-  // Get doc for contact update.
-  getDoc.onCall(1).callsArgWith(1, null, doc);
-  // Update doc : deleted contact
-  insertDoc.onCall(0).callsArgWith(1, null);
-  // Create person doc
-  createPerson.onCall(0).callsArgWith(1, null, { id: 'c'});
-  // Update doc : reset the contact field
-  updatePlace.onCall(0).callsArgWith(2, null);
-
-  migration.run(function(err) {
-    test.equals(err, undefined);
-    // Called once per place type.
-    test.equals(getView.callCount, 3);
-
-    test.equals(getDoc.callCount, 2);
-    test.equals(getDoc.firstCall.args[0], 'a');
-    test.equals(getDoc.secondCall.args[0], 'a');
-
-    // Contact was deleted, then reset.
-    test.equals(insertDoc.callCount, 1);
-    test.deepEqual(insertDoc.firstCall.args[0], docWithoutContact);
-    test.equals(updatePlace.callCount, 1);
-    test.equals(updatePlace.firstCall.args[0], 'a');
-    test.deepEqual(updatePlace.firstCall.args[1], { contact: 'c'});
-
-    // Contact was created.
-    test.equals(createPerson.callCount, 1);
-    test.deepEqual(createPerson.firstCall.args[0], {
-        name: 'name',
-        phone: 'phone',
-        place: 'a'
-      });
-
-    test.done();
-  });
-
-};
-
-exports['run still updates contact if facility has no parent'] = function(test) {
-  testContactUpdated(
-    test,
-    { _id: 'a', contact: { name: 'name', 'phone': 'phone'} },
-    { _id: 'a'});
-};
-
-exports['run still updates contact if facility has migrated parent'] = function(test) {
-  testContactUpdated(
-    test,
-    {
-      _id: 'a',
-      contact: { name: 'name', 'phone': 'phone'},
-      parent: {_id: 'b', contact: {_id: 'f'}}
-    },
-    {
-      _id: 'a',
-      parent: {_id: 'b', contact: {_id: 'f'}}
-    });
-};
-
-exports['run updates contact and parent'] = function(test) {
```

Comment:
This and above tests are either covered by integration tests, or aren't valid test cases anyway